.gitignore

```diff
@@ -125,6 +125,8 @@ packages/
 *project.lock.json
 jsconfig.json
 package-lock.json
+*.metaproj
+*.metaproj.tmp
 
```

Comment:
do we need an entry for msbuildproperties.g.json?

Reply:
I put it in `Generated Files` because it was the only "safe" place I could put it, relative to the project, that I wouldn't expect to be redirected.